### ISSUE TITLE
Add UI elements for filtering exhibits by tag

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/spotlight#1400]
@jcoyne @cbeer @jkeck This is to take advantage of the exhibit tags added in #1385. This is a pretty quick idea so we might need some additional discussion and design iteration but I wanted to get something out there before the winter break.

### Filter by exhibit tag

The idea is that the "All topics" link is the default, and any other existing exhibit tags are then shown as additional links to filter by that tag. Examples:

--
![all](https://cloud.githubusercontent.com/assets/101482/11904292/0913146a-a574-11e5-86d5-948433d1b80b.png)

--
![arts](https://cloud.githubusercontent.com/assets/101482/11904303/0ffdff74-a574-11e5-807d-e32c7b43b192.png)

--
![science](https://cloud.githubusercontent.com/assets/101482/11904307/13d30c48-a574-11e5-988d-adf2802775f4.png)

**Comment by cbeer** [comment]
:+1:

**Comment by ggeisler** [comment]
Additional thoughts now that I've seen that @jcoyne had added the feature for adding tags at the individual exhibit-level:

The new tags input on the Configuation > Settings page seems compatible with the Manage Tags page I suggested (above) at the Exhibits (super-admin) level, in that individual exhibit admins can add tags for their exhibit, but the super-admins can still act as editors because they can remove or change exhibit-provided tags on the Manage Tags page. From the Stanford use case perspective, I don't love the idea of individual exhibit admins creating tags unless we can give them a controlled vocabulary to select from, but I can see that this might make more sense in other use cases.

The filter approach I mocked up above works if the institution expects a relatively small set of tags to be used across all exhibits (again, this matches the Stanford use case). If another use case would have many tags (more than 7 or 8, say), I'm not sure the idea I proposed will work so well; it will just get ugly with many links wrapping into multiple rows. So if an institution having 10 or a dozen or more tags across their exhibits is a realistic use case, maybe we switch (based on a minimum number of unique exhibit tags) the horizontal list of links to a dropdown menu, aligned right at the top of the page, with a label ("Filter exhibits") and the dropdown containing an alphabetical list of tags.

I still like the horizontal list of links mocked up above for filtering as the default and think it will work for what might be the most common use case (a guess until we see how more Spotlight implementors use it), but I wanted to propose the idea of an alternative filtering control when there are many exhibit tags available for filtering.
